packages/aragon-wrapper/src/index.js

```diff
@@ -791,6 +777,35 @@ export default class Aragon {
         throw new Error(`No method named ${methodName} on ${destination}`)
       }
 
+      const methodABI = app.abi.find(
+        (method) => method.name === methodName
+      )
+      if (!methodABI) {
+        throw new Error(`${methodName} not found on ABI for ${destination}`)
+      }
+
+      let transactionOptions = {
+        gasPrice: defaultGasPrice,
+      }
+
+      // If an extra parameter has been provided, it is the transaction options if it is an object
+      if (methodABI.inputs.length + 1 == params.length && typeof params[params.length - 1] === 'object') {
+        const options = params.pop()
+        transactionOptions = { ...transactionOptions, ...options }
+      }
+
+      if (methodABI.inputs.length != params.length) {
+        throw new Error(`Incorrect number of parameters for function ${methodABI.name}. Expected ${methodABI.input.length}, provided ${params.length}`)
```

Comment:
`web3.eth.abi.encodeFunctionCall()` will throw in this case, so we might not need an explicit check here.